File: archive/count-pairs-whose-sum-is-less-than-target.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number}
 */
var countPairs = function(nums, target) {
  ans = 0
  for (let i = 0; i < nums.length; i++) {
    for (let j = i + 1; j < nums.length; j++) {
      if (nums[i] + nums[j] < target) {
        ans++
      }
    }
  }

  return ans
};

File: count-pairs-whose-sum-is-less-than-target.go
func countPairs(nums []int, target int) int {
    ans:=0

    for i:=0; i < len(nums); i++ {
        for j:=i+1; j < len(nums); j++ {
            if (nums[i] + nums[j] < target) {
                ans++
            }
        }
    }

    return ans
}